.github/workflows/release-nuget.yaml

```diff
@@ -15,7 +15,9 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 8.0.x
```

Comment:
a publish action should only use one version as it's a single event. The testing ones should build an array matrix